src/serlio/modifiers/RuleAttributes.cpp

```diff
@@ -57,6 +57,27 @@ std::wstring getNiceName(const std::wstring& fqAttrName) {
 
 } // namespace
 
+std::map<std::wstring,int> getImportOrderMap(const prt::RuleFileInfo* ruleFileInfo) {
+	std::map<std::wstring, int> importOrderMap;
+	int importOrder = 0;
+	for (size_t i = 0; i < ruleFileInfo->getNumAnnotations(); i++) {
+		const prt::Annotation* an = ruleFileInfo->getAnnotation(i);
+		const wchar_t* anName = an->getName();
+		if (!(std::wcscmp(anName, ANNOT_IMPORTS))) {
```

Comment:
By convention we explicitly write the int comparison (same for pointers btw). Please change the statement to `== 0`